src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -91,13 +91,32 @@ export class QuestionSubmissionFormComponent implements OnInit {
   allowedToHaveParticipantComment: boolean = false;
 
   constructor(private feedbackQuestionsService: FeedbackQuestionsService,
-              private feedbackResponseService: FeedbackResponsesService) {
+    private feedbackResponseService: FeedbackResponsesService) {
     this.visibilityStateMachine =
-        this.feedbackQuestionsService.getNewVisibilityStateMachine(
-            this.model.giverType, this.model.recipientType);
+      this.feedbackQuestionsService.getNewVisibilityStateMachine(
+        this.model.giverType, this.model.recipientType);
   }
 
   ngOnInit(): void {
+    this.sortRecepientsByName(this.model);
+  }
+
+  /**
+   * Sorts recepients of feedback by their name.
+   *
+   * Feedback recepients are initially sorted by their email address.
+   */
+  private sortRecepientsByName(model: QuestionSubmissionFormModel): void {
```

Comment:
Private methods can access the model directly using `this.model`, so there is no need to pass it as a parameter. This can be changed to a pameterless function.

Reply:
Will make the changes! I thought it would be easier in case `model` was to ever be changed, using it as a parameter would lead to less dependency